Scan/Rapoo_E9070/setup.cmake

```diff
@@ -0,0 +1,44 @@
+###| CMake Kiibohd Controller Scan Module |###
+#
+# Written by Jacob Alexander in 2014 for the Kiibohd Controller
+#
+# Released into the Public Domain
+#
+###
+
+
+###
+# Overrides
+#
+set ( MANUFACTURER "Input Club" PARENT_SCOPE )
```

Comment:
Input Club didn't make the keyboard, so you can either remove this or put another name here :D